src/components/header/menus.js

```diff
@@ -9,7 +9,11 @@ const Container = styled.div`
 `;
 
 const Button = styled.button`
-
+  font-family ${(props) => props.theme.mainFont};
+  font-size ${(props) => props.theme.fontSmall};
+  text-transform: uppercase;
+  background-color: none;
+  border: none;
```

Comment:
We should add `cursor: pointer;` to the buttons

Reply:
This would be a great component to move up to the `styles` directory (or similar), then we could import it in `src/components/header/logOut.js` as the log-out button is currently not styled

Reply:
Done. I also added a few colors from https://blueprintjs.com/docs/#core/colors to have a simple hover state